presto-thrift-connector/README.md

```diff
@@ -0,0 +1,21 @@
+Thrift Connector
+================
+
+Thrift Connector makes it possible to integrate with external storage systems without Presto code modification.
+
+In order to do it an external system needs to provide Thrift Nodes implementing a given Thrift service interface.
```

Comment:
I rewrote the paragraph and added a pointer to `PrestoThriftService` at the bottom.